src/plugins/codeStyle/diagnosticMessages.ts

```diff
@@ -199,5 +201,19 @@ export const messages = {
         source: 'bslint',
         message: `${CS} File should follow Roku broadcast safe color cert requirement`,
         range
+    }),
+    noAssocarrayFieldType: (range: Range, severity: DiagnosticSeverity) => ({
+        message: "'assocarray' node fields can cause performance problems",
```

Comment:
These error messages should be worded in such a way that says "don't do this thing". The explanation for _why_ should be described in the README.md. This aligns with all of our other messaging, which simply describes what is being flagged.

Let's change this and the `noArrayFieldType` messages to `"Avoid using field type 'assocarray'"` and  `"Avoid using field type 'array'"`

Reply:
messages have been edited